src/workloads/query/TimeSeriesLastpointHighCardinality.yml

```diff
@@ -0,0 +1,337 @@
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: CreateIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaAscTimeDesc   # We need to flip the direction for the lastpoint query
+              IndexPattern: *sortPatternMetaAscTimeDesc
+
+  # QuiesceActor will run in phase 15.
+
+  - Name: MetaDescTimeAsc
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [16]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaDescTimeAsc
+
+  - Name: DropIndexForMetaDescTimeAsc
+    Type: RunCommand
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [17]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: DropIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaAscTimeDesc
+
+  # QuiesceActor will run in phase 18.
+  ##################################################################################################
+
+  ##################################################################################################
+  # Following 4 actors including the hidden QuiesceActor in phase 20 run a lastpoint query with a
+  # sort and group on meta subfield ascending and time descending.
+  - Name: CreateIndexForMetaAscTimeAsc
+    Type: RunCommand
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [19]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: CreateIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc   # We need to flip the direction for the lastpoint query
+              IndexPattern: *sortPatternMetaDescTimeDesc
+
+  # QuiesceActor will run in phase 20.
+
+  - Name: MetaAscTimeAsc
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [21]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeAsc
+
+  - Name: DropIndexForMetaAscTimeAsc
+    Type: RunCommand
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [22]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: DropIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc
+  ##################################################################################################
+
+  - LoadConfig:
+      Path: *phasePath
+      Key: QuiesceActor
+      Parameters:
+        Active: [3, 5, 8, 10, 13, 15, 18, 20]
+        MaxPhases: *MaxPhases
+
+AutoRun:
+  - When:
+      branch_name:
+        $gte: v7.0
+      mongodb_setup:
+        $eq:
+          - standalone
```

Comment:
We've gotten rid of our regular standalone variants. Having this line will just mean it will only be running on the ARM and Intel 3-Node ReplSet 2023-11 variants.

Reply:
Apologies but I don't understand what you're suggesting here. Could you clarify on it so that I know what should be changed? I could not exactly correlate these names with the updated build variants.

These lines are exactly same as in the companion workload `TimeSeriesLastpoint.yml`. And I was able to see that this workload is enabled on perf-3-node-replSet-intel.intel.aws.2023-11 and perf-3-node-replSet.arm.aws.2023-11 and perf-3-node-replSet-all-feature-flags.arm.2023-11. I think these would be enough.

Reply:
Sorry the way I worded that was confusing 😅 We've removed our regular standalone variants ([SERVER-89499](https://jira.mongodb.org/browse/SERVER-89499)). Having `$eq: standalone, standalone-XXX` won't cause any issues, it just means it won't match an existing variant, and therefore won't run on any standalone variants. I just wanted to call this out so you weren't surprised that the test isn't running on standalone variants despite having this line in your code.

For clarity it's probably best to remove the `standalone` lines here.

(JFYI WRT the correlation between these lines and our variants - our variants are defined in [perf.yml](https://github.com/mongodb/mongo/blob/master/etc/evergreen_yml_components/variants/perf/perf.yml) and [perf_branching.yml](https://github.com/mongodb/mongo/blob/master/etc/evergreen_yml_components/variants/perf/perf_branching.yml). Each variant has a `mongodb_setup: XXX` line. In your Genny file if you write `AutoRun: When: mongodb_setup: $eq: replica` this means it will run on any variant that has `mongodb_setup: replica`)

Reply:
Ah~ makes sense now!! Thanks for detailed explanation. Really appreciate it! Removed `standalone(-*)` for clarity just like you suggested.